Cargo.toml

```
@@ -12,17 +12,19 @@ keywords = ["lv2", "audio", "dsp", "safe", "signal"]
 repository = "https://github.com/RustAudio/rust-lv2"
 
 [badges]
-travis-ci = { repository = "https://github.com/RustAudio/rust-lv2", branch = "master" }
+travis-ci = { repository = "RustAudio/rust-lv2", branch = "master" }
 maintenance = { status = "actively-developed" }
 
 [dependencies]
-lv2-atom = { path = "atom" }
-lv2-core = { path = "core" }
-lv2-midi = { path = "midi" }
-lv2-time = { path = "time" }
-lv2-units = { path = "units" }
-lv2-urid = { path = "urid" }
-lv2-state = { path = "state" }
+lv2-atom = "1.0.0"
+lv2-core = "1.0.0"
+lv2-midi = "1.0.0"
+lv2-time = "0.1.0"
+lv2-units = "0.1.0"
+lv2-urid = "1.0.0"
+lv2-state = "1.0.0"
+lv2-worker = { path = "worker"}
```

Once you've pulled from `develop`, you can use the version of the `worker` crate here and override the path to it in the `patch` section.
```suggestion
lv2-worker = "0.1.0"
```